pkg/controller/volume/persistentvolume/scheduler_binder_test.go

```diff
@@ -1376,23 +1552,23 @@ func TestBindPodVolumes(t *testing.T) {
 			shouldFail: true,
 		},
 		"binding-times-out": {
-			binding:    binding1aBound,
-			cachedPVs:  []*v1.PersistentVolume{pvNode1a},
-			cachedPVCs: []*v1.PersistentVolumeClaim{unboundPVC},
+			binding:    makeBinding(unboundPVC, pvNode1aBound),
+			initPVs:    []*v1.PersistentVolume{pvNode1a},
+			initPVCs:   []*v1.PersistentVolumeClaim{unboundPVC},
 			shouldFail: true,
 		},
 		"binding-fails": {
-			binding:    binding1bBound,
-			cachedPVs:  []*v1.PersistentVolume{pvNode1b},
-			apiPVs:     []*v1.PersistentVolume{pvNode1bBoundHigherVersion},
-			cachedPVCs: []*v1.PersistentVolumeClaim{unboundPVC2},
+			binding:    makeBinding(unboundPVC2, pvNode1bBound),
+			initPVs:    []*v1.PersistentVolume{pvNode1b},
+			initPVCs:   []*v1.PersistentVolumeClaim{unboundPVC2},
 			shouldFail: true,
 		},
 		"check-fails": {
-			binding:    binding1aBound,
-			cachedPVs:  []*v1.PersistentVolume{pvNode1a},
-			cachedPVCs: []*v1.PersistentVolumeClaim{unboundPVC},
-			delayFunc: func(t *testing.T, testEnv *testEnv, pod *v1.Pod, binding *bindingInfo, pvcs []*v1.PersistentVolumeClaim) {
+			binding:  makeBinding(unboundPVC, pvNode1aBound),
+			initPVs:  []*v1.PersistentVolume{pvNode1a},
+			initPVCs: []*v1.PersistentVolumeClaim{unboundPVC},
+			delayFunc: func(t *testing.T, testEnv *testEnv, pod *v1.Pod, pvs []*v1.PersistentVolume, pvcs []*v1.PersistentVolumeClaim) {
+				pvc := pvcs[0]
 				// Delete PVC
 				// Update pvc cache, fake client doesn't invoke informers
```

Comment:
can you update the rest of these tests to not directly update the internal cache since it's not needed anymore?